src/Components/login/Login.js

```diff
@@ -0,0 +1,182 @@
+import React, { useState, Fragment } from "react";
+import { Link } from "react-router-dom";
+
+import { Button, TextField, InputAdornment } from "@mui/material";
+import { makeStyles } from "tss-react/mui";
+import VisibilityIcon from "@mui/icons-material/Visibility";
+import { VisibilityOff } from "@mui/icons-material";
+
+import Image from "../image/ImageContainer";
+import "./styleLogin.scss";
+
+const styles = makeStyles()((theme) => {
+  return {
+    textFieldStyle: {
+      marginBottom: 14,
+    },
+  };
+});
+
+const Login = () => {
+  const [messageSignIn, setMessageSignIn] = useState({
+    text: "Enter your detalls below.",
+    messageColor: { color: "rgb(150, 157, 166)" },
+    borderBottonStyle: { borderBottom: "1px solid rgb(179, 175, 177)" },
+  });
+
+  const [showPassword, setShowPassword] = useState(false);
+  const [intoDashboard, setIntoDashboard] = useState(false);
+  const [textAvatar, setTextAvatar] = useState("");
+  const [userName, setUserName] = useState("");
+
+  const { classes } = styles();
+
+  const passwordAndUsernameNotMatchMessage = () => {
+    setMessageSignIn({
+      text: "Oops! That email and password combination is not valid.",
+      messageColor: { color: "rgb(237, 85, 151)" },
+      borderBottonStyle: { borderBottom: "1px solid rgb(237, 85, 151)" },
+    });
+  };
+
+  const UsernameNotExistMessage = () => {
+    setMessageSignIn({
+      text: "Oops! Username does not exist",
+      messageColor: { color: "rgb(237, 85, 151)" },
+      borderBottonStyle: { borderBottom: "1px solid rgb(237, 85, 151)" },
+    });
+  };
+
+  const logInFunction = (e) => {
+    e.preventDefault();
+    let intoDisboardSet = false;
+    let emailInfo = document.querySelector("#Email").value;
+    let passwordInfo = document.querySelector("#Password").value;
```

Comment:
`emailInfo` needs to come from the state dont use `document.querySelector` same for password